src/common/framework/test_group.ts

```diff
@@ -20,7 +20,7 @@ export interface RunCase {
 
 // Interface for defining tests
 export interface TestGroupBuilder<F extends Fixture> {
-  test(name: string): TestBuilderWithName<F, never>;
```

Comment:
This `never` usage of never (hah) really made sense, and I have considered several times trying to do something about it. Thanks for cleaning it up!